ucey-star/ID8
# frontend for authorization pages

## Description

Implemented the FE for the auth pages. Added components for the Google and gradient buttons, as well as input and password fields.

## Type of Change


## Testing Instructions
Go through the /signup, /login, and /email pages and check if they correspond to designs and test the FE functionality (fields responding to input)


## Screenshots (if applicable)
